apps/webClient/src/components/View/index.tsx

```diff
@@ -39,11 +62,19 @@ const useStyles = makeStyles(theme => ({
 
 const View: React.FC<Props> = ({ children }) => {
   const classes = useStyles();
+  let location = useLocation();
+
+  const derivePageBackgroundClass = (path: string) => {
+    if (path === "/") return classes.redPageWrapper;
+    if (path === "/our-work") return classes.greenPageWrapper;
+    if (path === "/about") return classes.bluePageWrapper;
+    if (path === "/lets-talk") return classes.yellowPageWrapper;
+  };
 
   return (
     <div className={classes.root}>
       <main className={classes.main}>
-        <Box className={classes.pageWrapper}>
+        <Box className={derivePageBackgroundClass(location.pathname)}>
```

Comment:
I think we might be overcomplicating this.

Can we revert the `classes.root` and `classes.main` css, get rid of the `Box`, and just put the result of `derivePageBackgroundClass` on the `classes.root` element ( using clsx ).  I think that is the simplest way.  Let me know if there's something I misunderstand.

Reply:
Looking at this again, I also think the `Container` element is a part of the issue as well, as it does not have height: 100%.

Let's get rid of the `pageContainer` class.  replace `main` with a `Container maxWidth` element and give it the `classes.main` class.